core/src/consensus/stake/mod.rs

```diff
@@ -846,12 +857,23 @@ mod tests {
         assert!(result.is_err(), "Cannot self-nominate without a sufficient balance");
     }
 
+    fn increase_term_id_until(state: &mut TopLevelState, term_id: u64) {
```

Comment:
If `on_term_close` handles increasing term id, this will be `term_close_until(...)`

Reply:
No. It doesn't close terms. It just increases term id on the metadata

Reply:
Is there a meaning to differentiate closing the term from increasing the term id?

Reply:
Yes. It is hard to create test cases if we close terms to set the states.

Reply:
Okay.